lib/options.json

```diff
@@ -246,6 +246,21 @@
         },
         {
           "type": "null"
+        },
+        {
+          "type": "object",
+          "properties": {
+            "basePort": {
+              "type": "number"
+            },
+            "highestPort": {
+              "type": "number"
+            },
+            "retry": {
```

Comment:
After testing and investigating, i think `retry` is unnecessary here, we have `process.env.DEFAULT_PORT_RETRY` constant so we can avoid option here, it is unnecessary. Also we should respect `process.env.DEFAULT_PORT_RETRY` when port is number/string/null.